lib/models/navigation/page_data/initial_page_data.dart

```diff
@@ -0,0 +1,26 @@
+library initial_page_data;
```

Comment:
the widget here (ie. the initial page) is actually called AuthOrMain but InitialPage seemed cleaner - maybe I should have renamed the widget... I'll see if it's a big change, there's already quite a few files

Reply:
lol thanks for the heads up, Initial page is really clear to me